### DISCUSSION TITLE
Can i set the minimum deployment target of macOS to 10.10?

### DISCUSSION THREAD

**Discussion by industriousonesoft** [outfoxx/PotentCodables#9] [Q&A] [answered]
Hi, thank you for creating such a piece of art. This project helps a lot.
Here, I has a doubt: Why the minimum deployment target of macOS was set to 10.12?

**Comment by kdubb** [comment, marked as answer]
Considering the pace of Apple platform adoption we only concern ourselves with the last 2-3 versions. It may work all the way back to 10.10 but I have not tried it.

**Comment by industriousonesoft** [comment]
Got it. Thanks for your prompt reply. I will try it myself. 😺
